Desktop adaptation

Danie12345 opened the pull request:
Finished full adaptation of project (up to the moment) for the desktop version.

Changed and added many styles, as well as restructured the html.

Danie12345 committed:
Fixed integration from collaborators

Fixed styling slight issue regarding the shadow box for the project cards.

Renamed a class for multiple hidden elements when changing from mobile to pc.

Danie12345 committed:
Fixed mobile to pc style issues

Top menu now fully responsive.

Headline section only missing media icons responsiveness.

Danie12345 committed:
Update index.css

Fixed all styling issues with both top menu and headline section. Both now fully responsive for most devices. Transitioning from size is also responsive.

Danie12345 committed:
Fixed style errors from previous commits

Fixed style errors introduced in previous commits where multiple areas would be affected by a same class, made them work separately.

Danie12345 committed:
Finished adding responsiveness to projects grid

Changed display flex for grid and remodeled the entire projects section via index.css, adding new classes in index.html in the process.

Added nice animations on project cards.

Fixed many previous bugs and new ones.

Created bug where "get my resume" button disappears on pc version. To be fixed in future udpates.

Danie12345 committed:
Fixed missing "get my resume" button

Fixed the bug where the "get my resume" button would not show up in pc version.

Danie12345 committed:
About me section responsive on pc, fixed bugs

About me section added responsiveness.

Fixed incomplete bug fix from last commit regarding the "get my resume" button.

Danie12345 committed:
Added responsiveness to form

Added styles for responsiveness to the form in index.css.

Danie12345 committed:
Update footer for pc version

Horizontal line at the bottom is now responsive for pc version, disappearing entirely.

Danie12345 committed:
Fixed caption for projects section

Fixed text not being aligned vertically modified properties to its selector in index.css.

thecodechaser commented on index.html:
```diff
@@ -54,41 +56,46 @@ <h3 class="h3-first">I'm a Software Developer</h3>
-                            See Project
-                        </div>
-                    </a>
+                <li class="grid-item multi-post-stories-outer grid-item-top-pc">
+                    <a href="" class="multi-post-image"><img src="Icons/img-placeholder1.png" alt=""></a>
+                    <div class="multi-post-stories multi-post-stories-column">
+                        <h3 class="multi-post-title h3-second">Multi-Post Stories</h3>
+                        <p class="multi-post-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required; 
+                            has been the industry's standard dummy text ever since the 1500s, when an unknown 
+                            printer took a standard dummy text.
+                        </p>
```
- [x] You did a great job on the styles but for this project, you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)template as much as possible. Please reduce the gap between (image and details) these elements and make sure it matches the design on the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/160229779-8c22b521-ed04-4afe-8db5-2ed029569c5b.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/160229788-cb7a7776-321e-4295-9888-633668e53e40.png)

thecodechaser commented on index.html:
```diff
@@ -245,7 +252,6 @@ <h3 class="project-title">Professional Art Printing Data</h3>
-                <li class="no-project"></li>
             </ul>
         </section>
         <section class="about-me decorative-background2">
```
- [x] Please make sure the about section background image is vertically centered as on the design. For that, you need to reduce the width of about section details.

### Your design

![image](https://user-images.githubusercontent.com/83803414/160230029-0f194bad-f50e-4bd0-a1cf-8750893eb6c5.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/160230041-58f9d003-e42b-49be-ba53-c3f46da51645.png)

thecodechaser reviewed (changes requested):
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Danie12345👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Danie12345 committed:
Fixed responsiveness on multi-post-stories section

Background image now better sticks to the template, as well as the text and buttons beside it.

Danie12345 committed:
Deleted unwanted empty space

Deleted empty space in index.html, line 248.

Danie12345 committed:
Fixed background image to be centered

Fixed background image for about me section to be centered, as well as the text occupying the desired space.

Changed background colors for the dark sections to be those in the template.

sja-thedude reviewed (approved):
## Project Approved 👍 

Hi @Danie12345, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![giphy](https://user-images.githubusercontent.com/70269507/159485323-11dfc0c5-bd53-4956-b55a-2fc42f7a3845.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
